parser/config.go

```diff
@@ -103,7 +104,8 @@ func readConfFile(file string) []byte {
 	conf, err := ioutil.ReadFile(file)
 
 	if err != nil {
-		fmt.Println("failed reading yaml file\n", err)
+		fmt.Println("Error: failed reading yaml file. SPDX document not generated.\n")
```

Comment:
The convention in go it to stick to lower case, i.e. "error:...", not a big deal for that though.
I think we should not omit the error, as those are not always so intuitive and in some use-cases might be really hard for the user to resolve it without any detail.